llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -150,6 +150,11 @@ def __init__(self, config: MPTConfig):
                     log.info(f'Removing bias ({module.bias}) from {module}.')
                     module.register_parameter('bias', None)
 
+                # For transformer engine
```

Comment:
do you still need the above code removing bias after your change?

Reply:
guarantee that bias gets nerf'd???

Reply:
I need it :(

Reply:
I don't know why I need it, but I tried without it and it failed